XPlat/TensorFlow.Lite/External-Dependency-Info.txt

```diff
@@ -0,0 +1,19 @@
+THIRD-PARTY SOFTWARE NOTICES AND INFORMATION
+Do not translate or localize
+
+Xamarin Components for NAME incorporates 
+third party material from the projects listed below. The original copyright 
+notice and the license under which Microsoft received such third party 
+material are set forth below.  Microsoft reserves all other rights not 
+expressly granted, whether by implication, estoppel or otherwise.
+
+########################################
+# BEGIN:     NAME
+# https://github.com/
```

Comment:
https://github.com/tensorflow/tensorflow/ is the GitHub project

https://github.com/tensorflow/tensorflow/blob/master/LICENSE is the license 

We probably need to look through here as well:
https://github.com/tensorflow/tensorflow/blob/master/ACKNOWLEDGMENTS

And maybe more yet too...